web/tests/test_chapters.py

```
@@ -0,0 +1,634 @@
+import datetime
+from typing import Any
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)